src/alignment.rs

```diff
@@ -259,7 +259,7 @@ impl Alignment {
                         y_pretty.push('-');
                     }
                     AlignmentOperation::Xclip(len) => {
-                        for k in x.iter().take(len) {
+                        for k in x.iter().skip(x_i).take(len) {
```

Comment:
Are you sure this doesn't break non-Custom `Alignment`s? Or maybe to be clearer, is there either (i) a test case that also covers this for for some non-custom `Alignment`? I could imagine, that the [special casing further up](https://github.com/rust-bio/rust-bio-types/pull/63/files#diff-bbf847ec5fe1856c9b87a5ed352a29e1e5f54d9e9481ca88573d0511cd8283e7L203) is the actual problem? But you dug into this code in more detail, so are probably more confident in answering those question.

Reply:
@dcroote Will you have time to have another look here? Otherwise I would probably move ahead with release `1.0.1` at #62 for now...

Reply:
Sorry for the delay- I'll need more time to flesh out the test cases so feel free to move ahead with the release

Reply:
Just a quick ping to see if there is any planned activity, here. If so, feel free to update the branch, first.